### PR TITLE
enhance(BottomSheet):Removed targetEvent and added clickOutsideToClose

### LINKED ISSUE #2201: [docs] md-bottom-sheet - targetEvent
Obviously documentation for `md-bottom-sheet` was copy-pasted from `md-dialog` since this is what it says for the option `targetEvent`:
> A click's event object. When passed in as an option, the location of the click will be used as the starting point for the opening animation of the the dialog.

Perhaps this needs to be fixed.
Also looking at how `md-bottom-sheet` is animated, it just comes from the bottom of the screen instead of how it's described in the docs. Perhaps option `targetEvent` could be removed all  together from `md-bottom-sheet`?

### PR DESCRIPTION
**Pull request by EladBezalel**
closes #2201 and #2895

### CODE REVIEW

File: src/components/bottomSheet/bottom-sheet.spec.js [hunk 1]
```diff
@@ -2,6 +2,52 @@ describe('$mdBottomSheet service', function() {
   beforeEach(module('material.components.bottomSheet'));
 
   describe('#build()', function() {
+    it('should not close when `clickOutsideToClose == true`', inject(function($mdBottomSheet, $rootElement, $material) {
```
Comment (jelbourn) [inline src/components/bottomSheet/bottom-sheet.spec.js]: Keep lines under 100 columns

**Comment by jelbourn** [comment]
LGTM other than that one comment

File: src/components/bottomSheet/bottom-sheet.spec.js (hunk 1 above)
Comment (EladBezalel) [inline src/components/bottomSheet/bottom-sheet.spec.js]: all of this page is more than 100 columns, should i fix it?

Comment (jelbourn) [inline src/components/bottomSheet/bottom-sheet.spec.js]: I generally wouldn't ask someone to change existing lines, but you can if you want.

**Pull request closed without merging**